extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -120,11 +144,29 @@ public function isValidHeroDataItem( $item, $heroData ) {
 	 * required for testing mobile template rendering
 	 * @return bool
 	 */
-	public function isWikiaMobile() {
+	public function isMobile() {
 		return \F::app()->checkSkin( 'wikiamobile' );
 	}
 
 	/**
+	 * @return bool
+	 */
+	public function isMercury() {
+		global $wgArticleAsJson;
+		
+		return !empty( $wgArticleAsJson );
+	}
+
+	/**
+	 * @return bool
+	 */
+	public function isMercuryExperimentalMarkupEnabled() {
```

Comment:
Similar to https://github.com/Wikia/app/pull/10121/files#diff-023d0e6389c7cd29ccc7a5bc2889afcaR154 - I recommend moving it outside of this clas